client/app/states/login/login.state.js

```diff
@@ -74,8 +74,11 @@ function StateController (exception, $state, Text, RBAC, API_LOGIN, API_PASSWORD
         Session.destroy()
       }
     })
-    .catch(() => {
-      exception.catch('Login failed, possibly invalid credentials.')
+    .catch((response) => {
+      if (response.status === 401) {
+        vm.credentials.password = ''
+        Notifications.message('danger', '', __('Incorrect username or password.'), false)
```

Comment:
If the response has a server message we should probably display that here

Reply:
We probably could but a couple of thoughts.  The message that comes back just says "Authentication failed".  At a minimum the message we display is a little more specific and it is also translated vs the raw message from the server.   Keep in mind, I didn't add this notification message, merely moved it from the factory up to the state that it pertains to.

Reply:
And normally I wouldn't press on this matter, but my 👃 is in a bz right now where not showing the full error message issssssss no bueno https://bugzilla.redhat.com/show_bug.cgi?id=1498984